js/web/lib/wasm/jsep/webgpu/ops/pool.ts

```diff
@@ -336,17 +338,13 @@ const globalPoolAttributes = {
   cacheKey: ''
 };
 
-export const parseGlobalAveragePoolAttributes = (attributes: Record<string, unknown>): AveragePoolAttributes => {
-  const format = attributes.format as FormatAttributes['format'];
-  return {format, ...globalPoolAttributes, cacheKey: format};
```

Comment:
Maybe just `return {format, ...globalPoolAttributes};`

Reply:
This can be removed, save one exported API.